charts/graphprotocol-indexer-service/templates/ingress.yaml

```diff
@@ -0,0 +1,37 @@
+{{- if .Values.ingress.enabled -}}
+{{-   if semverCompare ">=1.14-0" .Capabilities.KubeVersion.GitVersion -}}
+apiVersion: networking.k8s.io/v1beta1
+{{-   else -}}
+apiVersion: extensions/v1beta1
+{{-   end }}
+kind: Ingress
+metadata:
+  name: {{ include "graphprotocol-indexer-service.fullname" . }}
+  labels:
+    {{- include "graphprotocol-indexer-service.labels" . | nindent 4 }}
+  {{- with .Values.ingress.annotations }}
+  annotations:
+    {{- toYaml . | nindent 4 }}
+  {{- end }}
+spec:
+  {{- if .Values.ingress.tls }}
+  tls:
+    {{- range .Values.ingress.tls }}
+    - hosts:
+        {{- range .hosts }}
+        - {{ . | quote }}
+        {{- end }}
+      secretName: {{ .secretName }}
+    {{- end }}
+  {{- end }}
+  rules:
+    {{- range .Values.ingress.hosts }}
+    - host: {{ . | quote }}
+      http:
+        paths:
+          - path: /*
```

Comment:
Cause GKE Ingress routes only / (full-match) if `/` specified. So `/*` is more universal. And we need different endpoints for participating in testnet

Reply:
So this is cloud-dependent implementation, right?
WDYT about using nginx-ingress as an ingress controller?

Reply:
So this route should works fine on all clouds with their own L7 LBs. I think it would be easier to use cloud-specific LBs cause indexer operator wouldn't need to maintain ingress